src/main/java/co/omise/requests/Request.java

```diff
@@ -70,6 +73,22 @@ public RequestBody getPayload() {
         return payload;
     }
 
+    /**
+     * Gets payload (request params).
+     *
+     * @return the payload as a readable string
+     */
+    public  String getPayloadToString(){
+        try {
+            final Buffer buffer = new Buffer();
+            payload.writeTo(buffer);
+            return buffer.readUtf8();
+        }
+        catch (final IOException e) {
+            return "Unable to process payload";
```

Comment:
Can you help understand why are we returning a string instead of throwing an exception?

Reply:
Fixed